doc/Authoring_Plugins.md

```diff
@@ -188,3 +188,9 @@ Then you'll only have to refresh the tab of the `Development Host` instance, the
 
 Note: you may use watch mode as well.
 
+## API of plug-ins
+[Browse typedoc of plug-ins](/api/)
+
+## VS Code implementation
+Theia is providing VS Code API. Check the following link to get the current status of what is implemented.
+[Compare Theia vs VS Code API](https://che-incubator.github.io/vscode-theia-comparator/status.html)
```

Comment:
Is it based on d.ts files comparison? I know that we have something in theia.d.ts but they are not used or implemented, i.e. `FileSystemProvider` except read and write, nothing works. It is marked as green.

Reply:
but super cool idea

Reply:
@akosyakov yes it's only d.ts comparison, it doesn't check implementation

if it's green it doesn't mean it's compliant, but if it's red, surely it doesn't work